### PR TITLE
Add subkey support

### PR DESCRIPTION
**Pull request by dauglyon**
Adds support for errors with metadata subkeys, allowing for the highlighting of unit-column derived errors

### CODE REVIEW

**Review by Tianhao-Gu — APPROVED** [review]
Do you mind adding some comments in the release_note? Other than that, LGTM.

File: lib/sample_uploader/utils/importer.py [hunk 1]
```diff
@@ -495,6 +496,12 @@ def import_samples_from_file(
             e.sample_name = err_row_sample_names[e.row]
         if e.row==None and e.sample_name!=None and e.sample_name in err_sample_name_indices:
             e.row = err_sample_name_indices[e.sample_name]
+        if e.subkey:
+            for group in column_groups:
+                if e.subkey in group and e.key == group["value"]:
+                    e.column = err_key_indices[group[e.subkey]]
+                    break
+        e.column_name = err_col_keys[e.column]
```
Comment (slebras) [inline lib/sample_uploader/utils/importer.py:504]: Is `e.column` sometimes None?

**Review by slebras — APPROVED** [review]
Just one question about a potential edge case

File: lib/sample_uploader/utils/importer.py (hunk 1 above)
Comment (dauglyon) [reply lib/sample_uploader/utils/importer.py:504]: hmm, not in existing usecases but just in case I'll use .get :)

**Pull request merged**